website/docs/r/iam_trusted_profiles_link.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles_link"
+description: |-
+  Manages iam_trusted_profiles_link.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles_link
+
+Provides a resource for iam_trusted_profiles_link. This allows iam_trusted_profiles_link to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_iam_trusted_profiles_link" "iam_trusted_profiles_link" {
+  cr_type = "cr_type"
+  link = { "crn" : "crn", "namespace" : "namespace" }
+  profile_id = "profile_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `cr_type` - (Required, Forces new resource, String) The compute resource type. Valid values are VSI, IKS_SA, ROKS_SA.
+* `link` - (Required, Forces new resource, List) Link details.
+Nested scheme for **link**:
+	* `crn` - (Required, String) The CRN of the compute resource.
+	* `namespace` - (Required, String) The compute resource namespace, only required if cr_type is IKS_SA or ROKS_SA.
+	* `name` - (Optional, String) Name of the compute resource, only required if cr_type is IKS_SA or ROKS_SA.
```

Comment:
Name of the compute resource, only required if cr_type is IKS_SA or ROKS_SA.
as
The name of the compute resource, only required if `cr_type` is **IKS_SA** or **ROKS_SA**.